css/properties/custom-property.json

```diff
@@ -112,6 +112,109 @@
               "standard_track": true,
               "deprecated": false
             }
+          },
+          "safe-area-inset": {
```

Comment:
Would you mind expanding these to separate features for each keyword? (See my other comments first, however, to simplify the copy-and-paste.)

Reply:
Ah I see, so that they each show up individually in the compat tables. Makes sense.